From e2e723cfe96ce6ed6558ed7e8fbb44b2283297da Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 13 Sep 2018 06:57:28 -0700
Subject: [PATCH] Define MAX_HANDLE_SZ

MAX_HANDLE_SZ is described in name_to_handle_at() to contain maximum
expected size for a file handle

Upstream-Status: Submitted [https://www.openwall.com/lists/musl/2018/09/13/16]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 include/fcntl.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/fcntl.h b/include/fcntl.h
index 5c3defcb..6a8b6d49 100644
--- a/include/fcntl.h
+++ b/include/fcntl.h
@@ -166,6 +166,7 @@ struct f_owner_ex {
 };
 #define FALLOC_FL_KEEP_SIZE 1
 #define FALLOC_FL_PUNCH_HOLE 2
+#define MAX_HANDLE_SZ 128
 #define SYNC_FILE_RANGE_WAIT_BEFORE 1
 #define SYNC_FILE_RANGE_WRITE 2
 #define SYNC_FILE_RANGE_WAIT_AFTER 4
-- 
2.19.0

